examples/vue-class-component/package.json

```diff
@@ -4,7 +4,8 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "nuxt": "latest",
-    "vue-class-component": "^5.0.1"
+    "nuxt-class-component": "^1.0.3",
+    "vue": "~2.4.2"
```

Comment:
Can be remove after https://github.com/nuxt-community/nuxt-class-component/pull/3 merged and released

Reply:
I think you can remove it now @clarkdo :)

Reply:
@Atinux It may need a version bump and publish:smile

Reply:
@Atinux removed, can be merged after nuxt-class-component npm published